api/spec/controllers/mno_enterprise/jpi/v1/admin/organizations_controller_spec.rb

```diff
@@ -60,12 +64,48 @@ module MnoEnterprise
         it { subject; expect(data['organizations'].first['id']).to eq(organization.id) }
         it_behaves_like 'an unauthorized route for support users'
       end
+    end
+
+    describe 'GET #support_search' do
+      subject { get :support_search, params}
+      let(:admin_role) { :support }
+      let(:params) { {} }
+      let(:data) { JSON.parse(response.body) }
+
+      context 'with an invalid search' do
```

Comment:
In addition to the minimum search length, could you also add a case with invalid search terms (eg: `admin_role = admin`)?

Reply:
yep, agreed specs could be a bit more verbose.